### DISCUSSION TITLE
Changing interface language

### DISCUSSION THREAD

**Discussion by jupiter23** [hydrogen-music/hydrogen#1145] [Hydrogen on OS X]
Hi! I downloaded and installed 1.0.1 from sourceforge (Hydrogen-1.0.1.dmg for osx). I had an older version installed previously and replaced with this one. The interface is in French and I can't figure out how to switch back to English!

**Comment by trebmuh** [comment]
`menu -> options -> Préferences ->Général -> Langue ->` choose English, then restart Hydrogen.

**Reply by jupiter23** [reply]
Merci pour l'aide! Il ne semble pas y avoir de menu "Langue" dans cette interface.

![Screen Shot 2021-01-31 at 16 41 26](https://user-images.githubusercontent.com/2393241/106398860-4453ca00-63e3-11eb-946d-b7e302e990b6.png)

**Reply by trebmuh** [reply]
Oh, that probably need a more recent hydrogen than 1.0.1 then.

I don't know how it works on macOS, but on linux with a French locale, I can launch hydrogen in English with typing `LANG=C hydrogen` in a terminal.

**Reply by cme** [reply]
Yeah, the explicit language selection is in the fix for #929 which is in for I think 1.0.2, which we haven't made a release of yet.

The translation selection is driven purely by the system locale selection, which in macOS is set by the "Language and Region" settings in System Preferences.

![image](https://user-images.githubusercontent.com/768466/106400710-fe652900-6417-11eb-8975-16d8f318006f.png)

What languages & regions do you have set there, and in what order? The Qt translation library prefers an exact match of language+region over just a language match like English. So if you have French/France in preferred languages/regions list, Qt will select it even if it's further down the list than English.

**Reply by jupiter23** [reply]
I have English set as primary language

<img width="661" alt="Screen Shot 2021-01-31 at 19 55 21" src="https://user-images.githubusercontent.com/2393241/106403834-7aeb0e00-63fe-11eb-9241-a02ba71f2f91.png">

**Reply by jupiter23** [reply]
> Oh, that probably need a more recent hydrogen than 1.0.1 then.
> 
> I don't know how it works on macOS, but on linux with a French locale, I can launch hydrogen in English with typing `LANG=C hydrogen` in a terminal.

On osx LANG=C loaded the software in French too
